src/components/PageContent/PageContent.astro

```diff
@@ -57,6 +58,13 @@ const t = useTranslations(Astro);
 	.block {
 		display: block;
 	}
+	.sticky-nav {
+		width: 100%;
+		position: sticky;
+		z-index: 2;
+		top: 0;
+		background: var(--theme-bg-gradient);
+	}
```

Comment:
Thanks so much for this @Yan-Thomas! I’m going to try to get this merged TODAY 🤞

Your first issue is easy to address by extending the nav left a bit like you suggested:

```suggestion
		margin-inline-start: -2rem;
		padding-inline-start: 2rem;
	}
```

Taking a look now to see if I can figure out the sticky thing — probably something to do with a parent `position` value for something. Sticky is tricky!